apps/03TicTacToe.js

```diff
@@ -23,24 +23,84 @@ function printBoard() {
 
 function horizontalWin() {
     // Your code here
+    if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn)
```

Comment:
looks nice and clean, but we DEFINITELY want to wrap the bodies of our if/else statements in curly braces `{}`